src/container/srv_target.c

```diff
@@ -580,6 +580,7 @@ cont_start_agg(struct ds_cont_child *cont)
 }
 
 /* ds_cont_child *******************************************************/
+
```

Comment:
Whitespace only changes in this file.

Reply:
Yes, this is an intentional whitespace fix.

Reply:
I have a githook to auto-fix anything that checkpatch and codespell can resolve but it normally only affects files that are already modified.